Datadog.Trace.Security.sln

```diff
@@ -0,0 +1,399 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
I think we should probably create this as a solution filter rather than a new solution, given that we're planning on deleting the other slns!

Reply:
Fixed in 9e48d82b. 

@anna-git  It means that it now needs to be run as admin to be able to use the AspNetMvc5 sample.